cmd/lncli/main.go

```diff
@@ -563,3 +566,74 @@ func networkParams(ctx *cli.Context) (*chaincfg.Params, error) {
 		return nil, fmt.Errorf("unknown network: %v", network)
 	}
 }
+
+func waitUntilStatus(ctx *cli.Context, desiredState lnrpc.WalletState,
+	timeout time.Duration) error {
+	connectionRetryTicker := time.NewTicker(connectionRetryInterval)
+
+	timeoutChan := time.After(timeout)
+
+connectionLoop:
+	for {
+		conn := lnrpc.NewStateClient(getClientConn(ctx, true))
```

Comment:
I think we should leave the original log messages in, just print to the console, so the user knows what's happening.